### ISSUE TITLE
install bug with user

### ISSUE THREAD

**Issue by Flow86** [hyperbox/hyperbox#6]
Hi,

tried to install 0.0.12 and ran into this problem:

```
# sh hbox_server-latest-linux_amd64.run
Verifying archive integrity... All good.
Uncompressing Hyperbox Server  100%

#       # #       # ######### ######### #########  ########   ######  #       #
#       #  #     #  #       # #         #        # #       # #      #  #     #
#       #   #   #   #       # #         #        # #       # #      #   #   #
#########    # #    ######### ######### #########  ########  #      #    # #
#       #     #     #         #         #     #    #       # #      #   #   #
#       #     #     #         #         #      #   #       # #      #  #     #
#       #     #     #         ######### #       #  ########   ######  #       #

http://kamax.io/hbox/

Debian-based system detected
Found suitable java version (>= 1.6), continuing...
vbox was detected as a dedicated user. Use it? [Y/n] y
Will install to /opt/hboxd
chown: invalid group: 'vbox:vbox'
Failed to set permissions on install dir
An error occurred and the installation will now be cancelled. Check log file for more details: /var/log/hboxd-install.log
``` 

there is no group "vbox" - the group is named vboxusers.
the installer should either ask for the group, or look it up correctly?!

**Comment by maxidor** [comment]
Thank you for the report, it is indeed an assumption that shouldn't be made.
I will fix that in the install script. Meanwhile as a workaround, you can simply create the group manually before installing.

**Comment by maxidor** [comment]
Has been fixed in hyperbox/server trunk, will be part of 0.0.13

**Comment by maxidor** [comment]
For anyone interested by a trunk build with this fix while waiting for 0.0.13: https://kamax.io/hbox/download/hbox-0.0.12-4-g7926785-linux_amd64-server.run
